src/app/components/common/node-editor/editor.ts

```diff
@@ -41,6 +41,7 @@ class Connection<A extends Node, B extends Node> extends ClassicPreset.Connectio
 
 let editor = new NodeEditor<Schemes>();
 let area: AreaPlugin<Schemes, AreaExtra>;
+let history = new HistoryPlugin<Schemes, HistoryActions<Schemes>>({timing: 100});
```

Comment:
What is the meaning of 100 here?

Reply:
This 100ms is actions grouping time. There will be a single entry in history array for all the actions occurring within this time duration. So after pressing Ctrl+z all actions occurred within this time duration will be reverted at once. Default value for this is 200ms.